chandra_aca/aca_image.py

```diff
@@ -266,3 +304,94 @@ def centroid_fm(img, bgd=None, pix_zero_loc='center', norm_clip=None):
         raise ValueError("pix_zero_loc can be only 'edge' or 'center'")
 
     return row, col, norm
+
+
+class AcaPsfLibrary(object):
+    """
+    Access the ACA PSF library, whch is a library of 8x8 images providing the integrated
+    (pixelated) ACA PSF over a grid of subpixel locations.
+
+    :param filename: file name of ACA PSF library (default=built-in file)
+    :returns: AcaPsfLibrary object
+    """
+
+    def __init__(self, filename=None):
+        from astropy.table import Table  # Table is a somewhat-heavy import
+        psfs = {}
+
+        if filename is None:
+            filename = os.path.join(os.path.dirname(__file__), 'aca_psf_lib.dat')
```

Comment:
Should the PR have the update to setup.py (or whichever strategy is picked)  to just install this as package data?

Reply:
That was in the ToDo list and it is now done.

Reply:
Yes.  I wasn't sure if "Make sure data file gets installed" was a plan to copy the thing somewhere or to correctly add to the package install process.